.github/workflows/workflow.yml

```diff
@@ -37,14 +37,14 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         ghc: ["latest", "8.4.4"]
-        cabal: ["latest", "3.2.0.0", "3.4.0.0-rc4"]
+        cabal: ["latest", "3.2.0.0", "3.4.0.0"]
         expect-fail: [false]
         exclude:
           - os: windows-latest
-            cabal: "3.4.0.0-rc4"
+            cabal: "3.4.0.0"
```

Comment:
I actually have no idea what happens if you exclude windows with cabal 3.4 and then include it again.

That said, the include/exclude for windows here was intended to test cabal 3.4 so they can both probably be removed now that 3.4 is out "for real"

Reply:
Good catch! I've pushed an update which removes these.